luma/lcd/device.py

```diff
@@ -46,6 +46,46 @@
 __all__ = ["pcd8544", "st7735", "ht1621", "uc1701x", "st7567", "ili9341"]
 
 
+class GPIOBacklight:
+    def __init__(self, gpio, pin=18, active_low=True):
+        self._gpio = gpio
```

Comment:
can you add a docstring and document the parameters for this class? also add version marker.

Reply:
Pushed updated commits